src/WebdriverClassicDriver.php

```diff
@@ -762,7 +773,7 @@ protected function createWebDriver(): void
             throw new DriverException('Base driver has already been created');
         }
 
-        $this->webDriver = RemoteWebDriver::create($this->webDriverHost, $this->getDesiredCapabilities());
+        $this->webDriver = ($this->webDriverInstantiator)($this->webDriverHost, $this->desiredCapabilities);
```

Comment:
The callable invocation syntax you've used has worked since PHP 5.4. I wonder why the `call_user_func` is even needed.

P.S.
Not an issue with this PR.